content/tracing/faq/resource-trace-doesn-t-show-up-under-correct-service.md

```diff
@@ -29,6 +29,6 @@ An example of modifying the top level name for Python can be found below:
         # do something
 ```
 
-This function explicitly sets both the Service name and Top Level Name, being tornado-notification and tornado.notify respectively
+This function explicitly sets both the service name and Top Level Name, being tornado-notification and tornado.notify respectively
```

Comment:
Should there be a period at the end of this sentence?

Reply:
👌